internal/pkg/cli/env_show.go

```diff
@@ -74,9 +84,60 @@ func (o *showEnvOpts) Ask() error {
 
 // Execute shows the environments through the prompt.
 func (o *showEnvOpts) Execute() error {
+	if err := o.initEnvDescriber(o); err != nil {
+		return err
+	}
+	env, err := o.retrieveData()
+	if err != nil {
+		return err
+	}
+	if o.shouldOutputJSON {
+		data, err := env.JSONString()
+		if err != nil {
+			return err
+		}
+		fmt.Fprintf(o.w, data)
+	} else {
+		fmt.Println(env)
+		//} else {
+		//	fmt.Fprintf(o.w, env.HumanString())
+	}
+
 	return nil
 }
 
+// TODO: Move to describe/env.go as Describe()
+func (o *showEnvOpts) retrieveData() (*describe.Environment, error) {
+	env, err := o.storeSvc.GetEnvironment(o.ProjectName(), o.envName)
+	if err != nil {
+		return nil, fmt.Errorf("get environment: #{err}")
+	}
+	apps, err := o.storeSvc.ListApplications(o.ProjectName())
```

Comment:
Are we filtering on which Applications are actually deployed to a given environment here in any way?

Reply:
Oh, good point. This is now outdated here because I've moved this logic into Describe (https://github.com/aws/amazon-ecs-cli-v2/pull/858/files), but over there I have the same problem--straight ListApplications without filtering for the specific env. I saw this in ListPipelines in the `pipeline show ask` PR. The solution here is different, though, right?

Reply:
Update: got help on this from @iamhopaul123 (via @kohidave )!

Reply:
I will add this filter in my next PR so that it can go directly into Describe(), which doesn't exist on this branch/in this PR. For now, https://github.com/aws/amazon-ecs-cli-v2/issues/864.